### ISSUE TITLE
add lcFirst function

### ISSUE THREAD

**Issue by david-gang** [blakeembrey/change-case#18]
It would be nice to have a lcFirst function.
This is needed for example to convert an object name to a variable in angular.

For example i want to convert SearchService object into searchService variable.

It is easy to craft a function of the type:

```javascript
function firstCharLowercase(string) {
    return  string.charAt(0).toLowerCase() + string.slice(1);

}
```

but it would be nicer to have it here because already an ucfirst function exists :-)

**Comment by blakeembrey** [comment]
Makes sense to me. I'll add it tonight :smile: What about using `camelCase` though?

**Comment by blakeembrey** [comment]
Hey! Just released this with `v2.3.0` :+1:

**Comment by david-gang** [comment]
Thanks a lot :-)
